src/sidebar/components/test/MarkdownEditor-test.js

```diff
@@ -472,6 +471,26 @@ describe('MarkdownEditor', () => {
       typeInTextarea(wrapper, '@johndoe', 'Escape');
       assert.isFalse(wrapper.find('Popover').prop('open'));
     });
+
+    it('opens popover when clicking textarea and moving the caret to a mention', () => {
+      const text = 'text @johndoe more text';
+      const wrapper = createConnectedComponent({
+        text,
+        atMentionsEnabled: true,
+      });
+      const textarea = wrapper.find('textarea');
+      const textareaDOMNode = textarea.getDOMNode();
+
+      // Popover is initially closed
+      assert.isFalse(wrapper.find('Popover').prop('open'));
+
+      // Move cursor to overlap with the mention
+      textareaDOMNode.selectionStart = text.indexOf('@') + 1;
+      act(() => textareaDOMNode.dispatchEvent(new KeyboardEvent('click')));
```

Comment:
```suggestion
      act(() => textareaDOMNode.dispatchEvent(new MouseEvent('click')));
```

A generic `Event` would also be sufficient, but not `KeyboardEvent`.

Reply:
Oops! This is a copy-paste issue. Now I wonder how did this test pass at all 🤔